app/components/peopletable/peopletable.less

```diff
@@ -150,10 +150,11 @@
   bottom: 0;
   left: 0;
   box-sizing: border-box;
-  display: block;
+  // display: block;
```

Comment:
Better to remove this line.
Specially with the // comment style, sometimes there is some problems.